sensor_msgs/Image[] images1
sensor_msgs/Image[] images2
---
int32[] matches
